app/routes/lending/LendableObjectsList.css

```diff
@@ -0,0 +1,44 @@
+@import url('~app/styles/variables.css');
+
+.searchBar {
+  margin-bottom: 2rem;
+}
+
+.lendableObjectsContainer {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 2rem;
+}
+
+.lendableObjectCard {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 0;
+}
+
+.lendableObjectImage {
+  height: 15rem;
+  object-fit: cover;
+}
+
+.lendableObjectFooter {
+  display: flex;
+  justify-content: center;
+  color: var(--lego-font-color);
+  font-size: 1.1rem;
```

Comment:
??

Reply:
Always use variables.
Howver, a shouldn't update the header size. If so, you have picked the wrong header. The size is set by the global styling. Better to let every header everywhere look the same.

Reply:
Actually you shouldn't really need to specify color, font-size and font-weight at all if you just use the header tag instead of a paragraph for the title

Reply:
I would remove this class all together, turn the "footer" to a Flex component and wrap the title to a header tag

Reply:
Ah okay, will do